lib/realtime/api/channel.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Realtime.Api.Channel do
+  @moduledoc """
+  Defines the Channel schema
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{}
+
+  @schema_prefix "realtime"
+  schema "channels" do
+    field(:name, :string)
+    timestamps()
+  end
+
+  def changeset(channel, attrs) do
+    channel
+    |> cast(attrs, [:name, :inserted_at, :updated_at])
+    |> put_timestamp(:updated_at)
+    |> maybe_put_timestamp(:inserted_at)
```

Comment:
why is that necessary?

Reply:
Since we're not using Ecto but Postgrex directly this isn't done automatically as Ecto is the one handling timestamps

Reply:
Can the channel records be edited in the future? Also, the `realtime.subscription` table uses the `created_at` column to indicate the creation time. So it would be nice to keep consistency

Reply:
I think the `created_at` in `realtime.subscription` is filled by default

Reply:
can change but this does deviate from the ecto standard (timestamps() uses inserted_at and updated_at) 

I'm unsure if we will be able to change names or any new info we add it here, for now I will assume so.

subscriptions does it with a `created_at = now()` in the insert query.